src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -44,6 +45,8 @@ internal class GrpcWorker : IWorker
         private readonly IInvocationHandler _invocationHandler;
         private readonly IFunctionMetadataProvider _functionMetadataProvider;
 
+        internal static readonly Regex _frameworkDescriptionRegex = new Regex(@"^(\D*)");
```

Comment:
Should this be static? This Regex is only used for worker init response message handling use case. No reason to keep this in the memory. If we remove static and move closer to usage/method scope, GC can reclaim that memory.

Reply:
Doesn't have to be; was just trying to follow the practice of having this resolved early as suggested by Fabio every time I use regex xD I would make it not static and pass the variable to the `WorkerInitRequestHandler` method as a param instead? Or are you saying we should create this regex inside of `WorkerInitRequestHandler` - any perf concerns here?

Reply:
I think it might be best to keep it close to usage, within the method context. since this class is singleton, the private variable may stay alive I think.

Reply:
Do you know compiled regex may provide better perf? This gets used only once. But might be good to measure both compiled regex and non compiled.

Reply:
I'm sure that complied regex does provide better perf (Fabio and Nick have called this out in a couple of my PRs)

Reply:
Since `WorkerInitRequestHandler ` in theory only gets called once in the lifetime of a worker(?), I can move it inside the method scope as suggested. I think ideally it we do the precompiled regex but it might not be worth it xD